src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/HostWorkspace/LoadedProject.cs

```diff
@@ -126,9 +126,9 @@ private void AssetsFileChangeContext_FileChanged(object? sender, string filePath
         return _mostRecentFileInfo.TargetFramework;
     }
 
-    public ProjectSystemProject GetProjectSystemProject()
+    public ProjectId GetProjectSystemProjectId()
```

Comment:
A property `public ProjectId ProjectId => _projectSystemProject.Id;` would be fine here